modules/api/src/main/java/org/eclipse/xpanse/api/controllers/AdminServicesApi.java

```diff
@@ -109,17 +110,17 @@ public SystemStatus healthCheck() {
      * @return Returns list of cloud service provider.
      */
     @Tag(name = "Admin", description = "APIs for administrating Xpanse")
-    @Operation(description = "List cloud service provider.")
-    @GetMapping(value = "/csp", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "List cloud service providers.")
+    @GetMapping(value = "/csps", produces = MediaType.APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.OK)
-    @Secured({ROLE_ADMIN, ROLE_ISV, ROLE_USER})
-    public List<Csp> getCsps(
+    @Secured({ROLE_ADMIN, ROLE_CSP, ROLE_ISV, ROLE_USER})
+    public List<Csp> listCsps(
```

Comment:
I think it makes more sense to name it -> getActiveCsps
Probably also URI -> /csps/active

Reply:
Updated the API as suggested